docs/DeveloperGuide.md

```diff
@@ -294,17 +294,71 @@ The activity diagram shows the general sequence of steps when a user interacts w
 
 <br>
 
+## Implementation of Entity Commands
+Entity commands include `add`, `delete`, `find`, `clear` commands
+
+**Step 1**. The user types an `xyz` command in the `CommandBox`, followed by the type of entity `person` or `appt`, followed by appropriate arguments and prefixes.
```

Comment:
Would it be better if we split this sentence into two? 

Eg The user types an `xyz` command in the `CommandBox`, followed by the type of entity `person` or `appt`. This is followed by appropriate prefixes and arguments.